<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendWidget</class>
 <widget class="QWidget" name="FriendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="AvatarWidget" name="avatar">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QFrame" name="usernameFrame">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="unicorn::Label" name="username">
          <property name="text">
           <string notr="true">Username</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlayableItemWidget" name="radio">
          <property name="text">
           <string notr="true">PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="unicorn::Label" name="userDetails">
       <property name="text">
        <string notr="true">User details</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="trackFrame">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="unicorn::Label" name="lastTrack">
          <property name="text">
           <string notr="true">Artist - Title</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="equaliser">
            <property name="text">
             <string notr="true">e</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="unicorn::Label" name="timestamp">
            <property name="text">
             <string notr="true">Timestamp</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>unicorn::Label</class>
   <extends>QLabel</extends>
   <header>lib/unicorn/widgets/Label.h</header>
  </customwidget>
  <customwidget>
   <class>PlayableItemWidget</class>
   <extends>QPushButton</extends>
   <header>../Widgets/PlayableItemWidget.h</header>
  </customwidget>
  <customwidget>
   <class>AvatarWidget</class>
   <extends>QLabel</extends>
   <header>lib/unicorn/widgets/AvatarWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
